mix.exs

```diff
@@ -33,7 +33,7 @@ defmodule UeberauthCognito.MixProject do
   # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
-      {:excoveralls, "~> 0.13.4", only: :test},
+      {:lcov_ex, "~> 0.2", only: [:dev, :test], runtime: false},
```

Comment:
What's the minimum version of Elixir for LcovEx? I wonder if we should be using a `matrix` build to test that this package works with different versions of Elixir / Erlang?

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix

Reply:
1.9, but in the process of working on this I saw a warning for another dependency (`earmark_parser`) that required 1.10.

I wasn't aware of matrix jobs, though, that's an interesting idea!

Reply:
So thinking through the matrix build options idea, I think the main issue would be how to inject the relevant version numbers into the various files in which they're needed. It's probably feasible, but I'm not sure if it's worth the investment right now.